viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -85,6 +125,29 @@ export class IconCollection {
     };
   }
 
+  private computeProximityPoints(octree: IconOctree, iconSprites: InstancedIconSprite): BeforeSceneRenderedDelegate {
+    return ({ camera }) => {
+      const points =
+        this._proximityRadius < 1
```

Comment:
This condition seems odd to me, so if proximity radius is set to 0.5 then all points will be returned? I guess this should rather be when it is smaller than some epsilon?

Reply:
I reversed it. It is now `this._proximityRadius > 0`🤸‍♀️ .